packages/react-native-bpk-component-calendar/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-native-bpk-component-calendar",
+  "private": true,
+  "version": "0.0.17",
```

Comment:
Just out of interest where did `0.0.17` come from? Seems like a weird number for an unreleased component to have.

Reply:
It comes from the boilerplate 🤷‍♂️ 
https://github.com/Skyscanner/backpack-react-native/blob/master/packages/react-native-bpk-component-boilerplate/package.json#L4

Reply:
Done